src/main/java/de/retest/web/ScreenshotProvider.java

```diff
@@ -12,20 +14,33 @@
 
 	private static final int SCROLL_TIMEOUT = 100;
 	private static final boolean USE_DEVICE_PIXEL_RATIO = true;
+	private static final int WINDOW_WIDTH = 1200;
 
 	private ScreenshotProvider() {
 		// private constructor for util class
 	}
 
 	public static BufferedImage shootFullPage( final WebDriver driver ) {
+		final BufferedImage image;
 		if ( driver instanceof ChromeDriver ) {
-			return Shutterbug
+			image = Shutterbug
 					.shootPage( driver, ScrollStrategy.WHOLE_PAGE_CHROME, SCROLL_TIMEOUT, USE_DEVICE_PIXEL_RATIO )
 					.getImage();
+			return resize( image, WINDOW_WIDTH, image.getData().getHeight() / 2 );
 		} else {
-			return Shutterbug
+			image = Shutterbug
 					.shootPage( driver, ScrollStrategy.BOTH_DIRECTIONS, SCROLL_TIMEOUT, USE_DEVICE_PIXEL_RATIO )
 					.getImage();
+			return resize( image, WINDOW_WIDTH, image.getData().getHeight() );
 		}
 	}
+
+	private static BufferedImage resize( final BufferedImage image, final int width, final int height ) {
```

Comment:
This method belongs to `ImageUtils` in retest-model. Please add it there also, and mark this method with a `// TODO Remove with retest-model version 5.1.0 and use ImageUtils instead`.